### PR TITLE
Add Monitoring Google Cloud Samples

### PR DESCRIPTION
**Pull request by waprin**
Moves old samples into `api-client`.

### CODE REVIEW

File: monitoring/api/v3/api-client/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 google-api-python-client==1.6.1
-httplib2==0.10.3
+httplib2==0.9.2
```
Comment (jonparrott) [inline monitoring/api/v3/api-client/requirements.txt]: go ahead and remove all of these transitive dependencies.

File: monitoring/api/v3/cloud-client/snippets.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+from google.cloud.monitoring import (
```
Comment (jonparrott) [inline monitoring/api/v3/cloud-client/snippets.py]: `from google.cloud import monitoring`. Import modules not names, please.

File: monitoring/api/v3/cloud-client/snippets.py [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+        labels={
+            'status': 'successful',
+        }
+    )
+    client.write_point(metric, resource, 3.14)
+
+
+def list_time_series(client):
+    metric = 'compute.googleapis.com/instance/cpu/utilization'
+    print(list(client.query(metric, minutes=5)))
```
Comment (jonparrott) [inline monitoring/api/v3/cloud-client/snippets.py]: prefer a temporary variable with printing things out, and in general, it's better for a list/iterator to be shown using a for loop:

```python
results = client.query(metric, minutes=5)

for result in results:
    print(result)
```

File: monitoring/api/v3/cloud-client/snippets.py [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+        help=list_time_series_reduce.__doc__
+    )
+
+    read_time_series_aggregate = subparsers.add_parser(
+        'list-time-series-aggregate',
+        help=list_time_series_aggregate.__doc__
+    )
+
+    args = parser.parse_args()
+    client = Client(args.project_id)
```
Comment (jonparrott) [inline monitoring/api/v3/cloud-client/snippets.py]: We generally put the client construction into each snippet.

Comment (waprin) [inline monitoring/api/v3/cloud-client/snippets.py]: was basing on datastore but done

File: monitoring/api/v3/cloud-client/snippets.py (hunk 3 above)
Comment (waprin) [inline monitoring/api/v3/cloud-client/snippets.py]: done

File: monitoring/api/v3/cloud-client/snippets.py (hunk 2 above)
Comment (waprin) [inline monitoring/api/v3/cloud-client/snippets.py]: done

File: monitoring/api/v3/api-client/requirements.txt (hunk 1 above)
Comment (waprin) [inline monitoring/api/v3/api-client/requirements.txt]: done

File: monitoring/api/v3/cloud-client/snippets.py [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+    metric = 'compute.googleapis.com/instance/cpu/utilization'
+    query_results = client.query(metric, minutes=5).iter(headers_only=True)
+    for result in query_results:
+        print(result)
+
+
+def list_time_series_aggregate():
+    client = monitoring.Client()
+    metric = 'compute.googleapis.com/instance/cpu/utilization'
+    print(list(client.query(metric, hours=1).align(
```
Comment (jonparrott) [inline monitoring/api/v3/cloud-client/snippets.py]: Still printing lists here and in other methods.

File: monitoring/api/v3/requirements.txt [hunk 6]
```diff
@@ -1,10 +0,0 @@
-google-api-python-client==1.6.2
```
Comment (jonparrott) [inline monitoring/api/v3/requirements.txt]: Don't you need to keep at least the first requirement here?

Comment (waprin) [inline monitoring/api/v3/requirements.txt]: This is old file, new one is in api-client, Github just confused with the changes

File: monitoring/api/v3/cloud-client/snippets.py (hunk 5 above)
Comment (waprin) [inline monitoring/api/v3/cloud-client/snippets.py]: my bad, done

**Pull request merged**